#! /bin/sh /usr/share/dpatch/dpatch-run
## gcc-4.3.dpatch by Stefano Zacchiroli <zack@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: ensure sources build with gcc-4.3

@DPATCH@
diff -urNad trunk~/src/shared/Traits.hh.in trunk/src/shared/Traits.hh.in
--- trunk~/src/shared/Traits.hh.in	2003-01-14 12:41:55.000000000 +0100
+++ trunk/src/shared/Traits.hh.in	2008-05-01 15:45:39.000000000 +0200
@@ -26,6 +26,7 @@
  */
 
 #include <string>
+#include <cstring>
 
 #include "@DOM_NAMESPACE@Char.hh"
 
